e2e_samples/parking_sensors/adf/linkedService/Ls_AzureDatabricks_01.json

```diff
@@ -1,25 +1,20 @@
 {
 	"name": "Ls_AzureDatabricks_01",
-	"type": "Microsoft.DataFactory/factories/linkedservices",
 	"properties": {
 		"annotations": [],
 		"type": "AzureDatabricks",
 		"typeProperties": {
-			"domain": "https://australiaeast.azuredatabricks.net",
-			"accessToken": {
-				"type": "AzureKeyVaultSecret",
-				"store": {
-					"referenceName": "Ls_KeyVault_01",
-					"type": "LinkedServiceReference"
-				},
-				"secretName": "databricksToken"
-			},
+			"domain": "https://adb-7988601177969121.1.azuredatabricks.net",
```

Comment:
Same comment here for the domain and workspaceResourceId fields - can we avoid to commit that info on the repo?

Reply:
Same with above -- unfortunately comes with the git integration with ADF.